src/Riok.Mapperly/Helpers/SymbolExtensions.cs

```diff
@@ -14,8 +15,8 @@ internal static class SymbolExtensions
         typeof(Version).FullName
     );
 
-    internal static bool HasAttribute(this ISymbol symbol, INamedTypeSymbol attributeSymbol) =>
-        symbol.GetAttributes().Any(a => SymbolEqualityComparer.Default.Equals(a.AttributeClass, attributeSymbol));
+    internal static bool HasAttribute(this ISymbol symbol, INamedTypeSymbol attributeSymbol, WellKnownTypes types) =>
```

Comment:
If this is moved to the `WellKnownTypes` class, it could be simplified to `bool HasAttribute<TAttribute>(ISymbol symbol) where TAttribute : Attribute`